lib/db/schema/patch-042-043.sql

```diff
@@ -0,0 +1,20 @@
+CREATE PROCEDURE `prune_2` (IN pruneTokensMaxAge BIGINT UNSIGNED)
```

Comment:
naming nit: this value is not the same `pruneTokensMaxAge` as in the config file, it's an absolute `pruneTokensOlderThanDate`.  Worth renaming to avoid any potential confusion?